content/reference-rippled.md

```diff
@@ -10693,6 +10701,311 @@ The response follows the [standard format](#response-formatting), with a success
+                    "n9Ltz6ZxPRWTkqwBbpvgbaXPgm6GYCxCJRqFgNXhWVUebgezo28H",
+                    "n94D73ZKSUaTDCnUqYW5ugJ9fHPNxda9GQVoWA6BGtcKuuhozrD1"
+                ],
+                "pubkey_publisher":"ED58ED4AA543B524F16771F6E1367BAA220D99DCF22CD8CF7A11309E9EAB1B647B",
+                "seq":1,
+                "version":1
+            }
+        ],
+        "status":"success",
+        "trusted_validator_keys":[
+            "n94D73ZKSUaTDCnUqYW5ugJ9fHPNxda9GQVoWA6BGtcKuuhozrD1",
+            "n9Ltz6ZxPRWTkqwBbpvgbaXPgm6GYCxCJRqFgNXhWVUebgezo28H"
+        ],
+        "validation_quorum":2,
+        "validator_list_expires":"2017-Oct-13 14:56:00"
+    },
+    "status":"success"
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+The response follows the [standard format](#response-formatting), with a successful result containing the following fields:
+
+| `Field`                  | Type   | Description                              |
+|:-------------------------|:-------|:-----------------------------------------|
+| `listed_static_keys`     | Array  | Array of public keys for validators always eligible for inclusion in the trusted list. |
+| `publisher_lists`        | Array  | Array of publisher list objects.         |
+| `trusted_validator_keys` | Array  | Array of public keys for currently trusted validators. |
+| `validation_quorum`      | Number | Minimum number of trusted validations required to validate a ledger version. Some circumstances may cause the server to require more validations. |
+| `validator_list_expires` | String | Either the human readable time when the current validator list will expire or the string `unknown` if the server has yet to load a published validator list. |
+
+Each member of the `publisher_lists` array is an object with the following fields:
+
+| `Field`            | Type             | Description                              |
+|:-------------------|:-----------------|:-----------------------------------------|
+| `available`        | Boolean          | If `false`, the validator keys in `list` may no longer be supported by this publisher. |
+| `expiration`       | String           |  Either the human readable time when this published list will expire or the string `unknown` if the server has yet to load a list for this publisher.      |
+| `list`             | Array            | Array of published validator keys.   |
+| `pubkey_publisher` | String           | Hex encoded public key of publisher. |
+| `seq`              | Unsigned Integer | The sequence number of this published list. | 
+| `version`          | Unsigned Integer | The version of the list format.      |
+
+#### Possible Errors ####
+
+* Any of the [universal error types](#universal-errors).
+
+
+## validator_sites ##
+[[Source]<br>](https://github.com/ripple/rippled/blob/master/src/ripple/rpc/handlers/ValidatorSites.cpp "Source")
+
+The `validator_sites` command returns status information of sites serving validator lists. [New in: rippled 0.90.0]
```

Comment:
Please change `[New in: rippled 0.90.0]` to `[New in: rippled 0.90.0][]` for greater Markdown compatibility